manifests/k/KatjaHoffmann/TViewer/2.3.0.5/KatjaHoffmann.TViewer.locale.de-DE.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: KatjaHoffmann.TViewer
+PackageVersion: 2.3.0.5
+PackageLocale: de-DE
+Publisher: Katja Hoffmann
+PackageName: TViewer
+License: proprietary
+Copyright: Copyright © 2006-2021 Katja Hoffmann
+ShortDescription: TViewer Setup

```

Comment:
Please provide a better description of what TViewer does

Reply:
TViewer is a program to visualize the results of german equestrian competions based on the data of the Toris (it's the program, used to manage equestrian competitions program.